drivers/renesas/rcar/ddr/ddr_b/boot_init_dram.c

```diff
@@ -0,0 +1,4467 @@
+/*
```

Comment:
The checkpatch errors need to be corrected. Perhaps `--fix` can rectify most of the problems since they are whitespace issues.

Reply:
I think it is unrealistic that I will be able to fix the ~1600 errors being reported (the different tools generate more errors themselves in a catch22 sequence). Would it be enough if I were to focus on the platform/ and skim over the drivers - particularly the larger ones like DDR ?